[PLAT-14912] docs warning for install root as subdirectory (#23470)
yugabyte/yugabyte-db: docs/content/preview/yugabyte-platform/install-yugabyte-platform/migrate-replicated.md

    Note that the `installRoot` (by default `/opt/ybanywhere`) in `yba-ctl.yml` needs to be different from the Replicated Storage Path (by default `/opt/yugabyte`). If they are set to the same value, the migration will fail. You can delete `yba-ctl.yml` and try again.

    {{< warning title="Install Root" >}}

> The `installRoot` must **_not_** be a subdirectory of the Replicated Storage Path (`/opt/yugabyte` and `/opt/yugabyte/ybanywhere`) or upon completion of `replicated-migrate finish` all data will be lost while cleaning up Replicated.

Reviewer: ```suggestion
The `installRoot` must _not_ be a subdirectory of the Replicated Storage Path (`/opt/yugabyte` and `/opt/yugabyte/ybanywhere`). If `installRoot` is in the Replicated Storage Path, `replicated-migrate finish` will delete all your data while cleaning up Replicated.
```

>     The `installRoot` must ___not___ be a subdirectory of the Replicated Storage Path (`/opt/yugabyte` and `/opt/yugabyte/ybanywhere`) or upon completion of `replicated-migrate finish` all data will be lost while cleaning up Replicated.

Reviewer: ```suggestion
The `installRoot` must ___not___ be a subdirectory of the Replicated Storage Path (`/opt/yugabyte` and `/opt/yugabyte/ybanywhere`) or upon completion of `replicated-migrate finish` all data will be lost while cleaning up Replicated.
```
